deploy/25_role.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I guess we need this clusterrole to get and label nodes, right?

Reply:
To be able to see all nodes, we need clusterrole

Reply:
In that case we only need those permissions

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: node-reader
rules:
- apiGroups:
  - ""
  resources:
  - nodes
  verbs:
  - get
  - list
  - patch
```

Reply:
OK. reduce the permission scope to your suggestion